packages/web-components/src/menu/menu.styles.ts

```diff
@@ -28,4 +29,24 @@ export const styles = css`
   ::slotted([popover]:not(:popover-open)) {
     display: none;
   }
+
+  :host([split]) {
+    display: inline-flex;
+  }
+
+  :host([split]) ::slotted([slot='primary-action']) {
+    border-inline-end: ${strokeWidthThin} solid ${colorNeutralStroke1};
```

Comment:
I wonder if `border-inline-end: ${strokeWidthThin} solid currentColor;` would work, if it does, you probably don't need the rule for `appearance=primary` below.

Reply:
currentColor is the wrong color for the divider

<img width="803" alt="image" src="https://github.com/user-attachments/assets/f0b2bc44-838d-41be-aef3-70f2cb5f8322">

Reply:
Ah i see, never mind then.